lib/tools/convert_time.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function convert(time) {
+  var winTicks = 10000000;
+  var uEpoch = 11644473600;
+  var unixTime = time / winTicks - uEpoch;
+  return new Date(unixTime * 1000);
+};
```

Comment:
Any documentation on this?

Reply:
https://stackoverflow.com/questions/6161776/convert-windows-filetime-to-second-in-unix-linux